third_party/xla_client/BUILD

```diff
@@ -360,7 +359,6 @@ cc_library(
         "@com_google_absl//absl/types:span",
         "@org_tensorflow//tensorflow/compiler/xla:statusor",
         "@org_tensorflow//tensorflow/compiler/xla:types",
-        "@org_tensorflow//tensorflow/core:lib",
```

Comment:
Change this to @org_tensorflow//tensorflow/tsl/platform:hash

Reply:
tried and passed BUILD, but failed on CPU test